source/retryable-writes/retryable-writes.rst

```diff
@@ -137,21 +137,20 @@ check the user’s command document to determine if it is a write, nor check
 whether the server is new enough to support a transaction ID for the command.
 The method should simply send the user’s command document to the server as-is.
 
-This specification does not affect write commands executed outside of a
-DriverSession or within a DriverSession where retryable writes have not been
-enabled.
+This specification does not affect write commands executed in an application
+where retryable writes have not been enabled.
 
 Constructing Write Commands
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-When constructing any write command that will be executed within a DriverSession
+When constructing any write command that will be executed within a MongoClient
 where retryable writes have been enabled, drivers MUST increment the transaction
 number for the corresponding server session and include the server session ID
 and transaction number in top-level ``sessionId`` and ``txnNum`` fields,
-respectively. The ``sessionId`` value is an opaque BSON type and MUST be
-obtained from the ``startSession`` command (as discussed in the Driver Session
-specification). The ``txnNum`` value MUST be an unsigned integer and the server
-will accept 32-bit (BSON type 0x10) or 64-bit (0x12) values.
+respectively. The ``sessionId`` value is an opaque BSON type (as discussed in
```

Comment:
They had a discussion about it in one of the documents, IIRC. They know that other global command parameters are $-prefixed and have decided that these parameters are not.